src/bo4e/bo/geschaeftsobjekt.py

```diff
@@ -28,6 +33,7 @@ class Geschaeftsobjekt:
 class GeschaeftsobjektSchema(Schema, JavaScriptMixin):
     """
     This is an "abstract" class.
+    Inherits from Schema and JavaScriptMixin.
     All business objects are inherited from this class.
```

Comment:
```suggestion
    All business objects do inherit from this class.
```
Für mich ist das eher aktiv. Aber das ist Kleinkrämerei.